src/Scientist/IObservationPublisher.cs

```diff
@@ -8,10 +8,10 @@ namespace GitHub
     public interface IObservationPublisher
```

Comment:
Yeah I also updated the default implementation of the interface to be `InMemoryResultPublisher`.